src/harness/testcases/WINNF_FT_S_REG_testcase.py

```diff
@@ -423,7 +423,49 @@ def test_WINNF_FT_S_REG_5(self):
     for x in range(1, 4):
       self.assertEqual(
           response['registrationResponse'][x]['response']['responseCode'], 102)
+  
+  @winnforum_testcase
+  def test_WINNF_FT_S_REG_8(self):
+    """Invalid REG-Conditional parameters in Array Registration Request (responseCode 103)
+
+    The response should be SUCCESS for the first CBSD,
+    FAILURE 103 for the second and third CBSDs.
+    """
+
+    # Load devices
+    device_1 = json.load(open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_3 = json.load(open(os.path.join('testcases', 'testdata', 'device_e.json')))
+
+    # Inject FCC IDs
+    self._sas_admin.InjectFccId({'fccId': device_1['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_2['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_3['fccId']})
 
+    # Inject User IDs
+    self._sas_admin.InjectUserId({'userId': device_1['userId']})
+    self._sas_admin.InjectUserId({'userId': device_2['userId']})
+    self._sas_admin.InjectUserId({'userId': device_3['userId']})
+
+    # Device 2 out-of-range or the wrong type azimuth
+    device_2['installationParam']['antennaAzimuth'] = -1
+
+    # Device 3 out-of-range, or the wrong Type value for latitude.
+    device_3['installationParam']['latitude'] = 91.0
+
+    # Register the devices
+    request = {'registrationRequest': [device_1, device_2, device_3]}
+    response = self._sas.Registration(request)['registrationResponse']
+
+    # Check registration response
+    # valid cbsdId and responseCode 0 for 1st cbsd
+    self.assertTrue('cbsdId' in response[0])
```

Comment:
following a discussion with Kate in an email I sent to dev test list(Question about "Validation" of CbsdId), we may need to check the length in each test that asks us to verify that a valid CBSD ID is returned.
do you agree with that?

Reply:
I agree with Kate, maybe we cann't do more than checking the length of CBSD ID.